tests/shell/Makefile

```diff
@@ -12,4 +12,10 @@ BOARD_BLACKLIST += chronos
 
 TEST_ON_CI_WHITELIST += all
 
+# In order to properly test and debug the shell it is better to use a terminal
+# program that does not modify the input and output streams and has no buffering.
+ifneq ($(BOARD), native)
+  RIOT_TERMINAL?=miniterm.py
```

Comment:
please put spaces around `?=` look nicer and (more) inline with the rest of our makefiles stuff.

Reply:
```suggestion
  RIOT_TERMINAL ?= miniterm.py
```